README.md

```diff
@@ -71,6 +71,14 @@ Keep these goals in mind:
 You can use standard libraries to whatever extent is appropriate. In particular, you should not find it necessary to implement your own complete JSON parser.
 
 
+### Resources
```

Comment:
This section probably obsoletes the last sentence preceding it (which was a softer attempt at the same goal).